doc/sec_paper/AbelJacobiMap.tex

```diff
@@ -73,18 +72,17 @@
       \OB = \left(\int_{β_i}ω_j\right)_{1\leq i,j\leq g}
   \end{equation}
   and call {\em big period matrix} the concatenated matrix \\
-  \todo (CN) without comma? $(\OA \, \OB)$? $\in \C^{g \times 2g}$ or $\C^{2g \times g}$?
   \begin{equation}
-      \Omega = (\OA,\OB)
+      \Omega = (\OA \,\OB) \in \C^{g\times 2g}
   \end{equation}
   such that $\Lambda = \Omega\Z^{2g}$.
 
-  If one takes as basis of differentials the dual basis of
+  If one takes as basis of differentials the dual basis of\,,
```

Comment:
I have a question: why is your tex filled with small spaces like that? is it your editor? I think we should remove them all.